rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/ProductVersionEndpoint.java

```diff
@@ -114,6 +114,23 @@ void update(
             @Parameter(description = PV_ID) @PathParam("id") int id,
             @NotNull ProductVersion productVersion);
 
+    @Operation(summary = "Patch an existing product version.",
+            responses = {
+                    @ApiResponse(responseCode = SUCCESS_CODE, description = SUCCESS_DESCRIPTION,
```

Comment:
Should we use here (and on other patch specific) this?
```suggestion
                    @ApiResponse(responseCode = ENTITY_UPDATED_CODE, description = ENTITY_UPDATED_DESCRIPTION,
```

Reply:
According to the [guidelines](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH), if there is a response body, the code should be 200.